Packages/com.unity.inputsystem/InputSystem/Editor/Debugger/InputDebuggerWindow.cs

```diff
@@ -370,6 +370,7 @@ private static class Contents
             public static readonly GUIContent copyDeviceDescription = new GUIContent("Copy Device Description");
             public static readonly GUIContent copyLayoutAsJSON = new GUIContent("Copy Layout as JSON");
             public static readonly GUIContent createDeviceFromLayout = new GUIContent("Create Device from Layout");
+            public static readonly GUIContent generateCodeFromLayout = new GUIContent("Generate Precompiled Layout");
```

Comment:
We should look at localizing these in the future.

Reply:
Yup, indeed.